charts/workflow-controller/templates/deployment.yaml

```diff
@@ -2,12 +2,13 @@ apiVersion: extensions/v1beta1
 kind: Deployment
 metadata:
   name: {{ template "fullname" . }}
+  namespace: kube-system
```

Comment:
I didn't put initially the namespace, in order to allow the user to chose in which namespace he want to run the workflow controller.
`helm install  --namespace="kube-system" charts/workflow-controller`

Is it ok for you?

Reply:
Sure, `helm` can install resources into target namespace. I just think that CRD controller should always be in `kube-system` namespace. There are no too many examples of CDR out there, but all, that I saw so far, are deployed into `cube-system`. Maybe to avoid accidental remove of CRD controller when deleting namespace (`kube-system` is kind-of "system" namespace)

Do you see any reason to install CRD controller into other namespace than `kube-system`?

Reply:
It depends on the permission model that you want to use with namespace and operation. 
In some company (for example mine) peoples responsible for the k8s cluster support may not be the same peoples responsible for the workflow-controller, and so the workflow-controller will have a dedicated namespace.

Reply:
I see. Will update.